vembrane/__init__.py

```diff
@@ -23,32 +30,40 @@
 }
 
 
-def get_annotation_keys(header: VariantHeader) -> List[str]:
+def get_annotation_keys(header: VariantHeader,) -> List[str]:
```

Comment:
did black introduce the trailing comma? i.e. was there a style change?

Reply:
clickbait: the linter hates THESE commas